lib/packages/crypto/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@burst/crypto",
+  "version": "1.0.0",
+  "description": "Cryptographic functions for building Burstcoin apps.",
+  "contributors": [
+    "ohager",
+    "blankey1337"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/burst-apps-team/phoenix"
+  },
+  "engines": {
+    "node": ">=10"
+  },
+  "license": "GPL-3.0",
+  "main": "./src/index.ts",
+  "private": false,
+  "jest": {
+    "transform": {
+      "^.+\\.tsx?$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+    },
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx",
+      "json"
+    ]
+  },
+  "devDependencies": {
+    "@types/jest": "^23.3.12"
+  },
+  "scripts": {
+    "test": "jest",
+    "doc": "./node_modules/.bin/esdoc"
+  },
+  "dependencies": {
+    "@burst/http": "file:../http",
+    "@burst/core": "file:../core",
```

Comment:
I think this is wrong but couldn't get it to work using real version numbers (`1.0.0`) without `lerna publish` which I dont think we are ready to do yet.